apps/amo/urlresolvers.py

```diff
@@ -214,6 +215,26 @@ def linkify_only_full_urls(attrs, new=False):
     return attrs
 
 
+URL_RE = re.compile(r'\bhttps?://([a-z0-9-]+\.)+({0})/[^\s<>&"\x27]*'
+                    .format('|'.join(bleach.TLDS)))
+
+
+def linkify_escape(text):
+    """Linkifies plain text, escaping any HTML metacharacters already
+    present."""
+    # Bleach does a monumentally bad job at this. If we pass it escaped HTML
+    # which contains any URLs (&lt;div&gt;http://foo.bar/&lt;div&gt;),
+    # we get back HTML (<div><a href="http://foo.bar/</div>).
+    #
+    # So just stick to search-and-replace. We can hardly do a worse job than
+    # Bleach does.
+    def linkify(match):
+        # Parameters are already escaped.
+        return u'<a href="{0}">{0}</a>'.format(match.group(0))
```

Comment:
Yeah, normally, but these come from the validator, so they should be trusted URLs, and `rel="external"` is not helpful.